paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -650,29 +660,24 @@ def humanize_error(error):
         return 'utilization within thresholds'
 
 
-def get_configs_of_services_to_scale(cluster, soa_dir=DEFAULT_SOA_DIR):
-    services = get_services_for_cluster(
-        cluster=cluster,
-        instance_type='marathon',
-        soa_dir=soa_dir,
-    )
+def get_configs_of_services_to_scale(cluster, soa_dir=DEFAULT_SOA_DIR, services=None):
+    if not services:
```

Comment:
I would prefer that this part (fetching the list of services) be handled higher up (heck, main?) and not have this function do it

Reply:
Yeah, I see what you mean, let me have a crack at that